scripts/zkai_native_seq32_attention_mlp_adjacent_probe_b_bucket_attribution_gate.py

```
@@ -0,0 +1,737 @@
+def accounting_rows_by_path() -> dict[str, dict[str, Any]]:
+    data, raw = read_json_object(ACCOUNTING_PATH, "adjacent seed-sweep accounting", MAX_ACCOUNTING_JSON_BYTES)
+    digest = sha256_bytes(raw)
+    if digest != EXPECTED_ACCOUNTING_SHA256:
+        raise AdjacentProbeBBucketAttributionGateError("accounting digest drift")
+    if data.get("schema") != "zkai-stwo-local-binary-proof-accounting-cli-v1":
+        raise AdjacentProbeBBucketAttributionGateError("accounting schema drift")
+    rows: dict[str, dict[str, Any]] = {}
+    for row in data.get("rows", []):
+        path = row.get("evidence_relative_path")
+        if not isinstance(path, str):
+            raise AdjacentProbeBBucketAttributionGateError("accounting row path missing")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)